tests/test.rs

```diff
@@ -413,7 +410,7 @@ mod sync {
     }
 
     #[test]
-    #[cfg(not(miri))] // leaks memory
+    #[cfg_attr(miri, ignore)] // leaks memory
```

Comment:
Would t be useful for you to have a flag in Miri to ignore memory leaks? That would ignore leaks for all tests, but it would enable these tests to run.

Reply:
I'd say in this case that's not really a leak, so ideally miri shouldn't flag it as such. But in the meantime, ignore-leaks flag would be useful, yeah!